packages/perseus-editor/src/components/interactive-graph-settings.tsx

```diff
@@ -545,112 +559,139 @@ class InteractiveGraphSettings extends React.Component<Props, State> {
                     </div>
                 </div>
 
-                <div className="image-settings">
-                    <div>Background image:</div>
-                    <div>
-                        <label htmlFor="bg-url">Url:</label>
-                        <input
-                            id="bg-url"
-                            type="text"
-                            className="graph-settings-background-url"
-                            ref={this.bgUrlRef}
-                            value={this.state.backgroundImage.url || ""}
-                            onChange={(e) => {
-                                const image = _.clone(
-                                    this.props.backgroundImage,
-                                );
-                                image.url = e.target.value;
-                                this.setState({backgroundImage: image});
-                            }}
-                            onKeyPress={this.changeBackgroundUrl}
-                            onBlur={this.changeBackgroundUrl}
+                <LabeledRow
+                    label="Background image URL:"
+                    style={styles.resetSpaceTop}
+                >
+                    <input
+                        type="text"
+                        className={css(styles.backgroundUrlInput)}
+                        ref={this.bgUrlRef}
+                        value={this.state.backgroundImage.url || ""}
+                        onChange={(e) => {
+                            const image = _.clone(this.props.backgroundImage);
```

Comment:
I think you swapped this `_.clone()` in other places for a simple object spread.